### PR TITLE
Adding sample fabfile, travis ci and MIT free to use LICENSE

### PR DESCRIPTION
**Pull request by chowmean**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+
+# command to install dependencies
+install: 
+  - "pip install safety"
+  - "pip install fab-polish"
```
Comment (adityav-verma) [inline .travis.yml:8]: The current version of `fab-polish` isn't python3 compatible.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
```
Comment (adityav-verma) [inline .travis.yml:3]: flask-app is built on python3.

File: fabfile.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from fabric.api import local, settings, abort
+from fabric.contrib.console import confirm
+from fabpolish import polish, sniff, local, info
+from fabpolish.contrib import find_merge_conflict_leftovers
+
+@sniff(severity='major', timing='fast')
+def composer_security_check_npm():
+    """Requires nsp in package"""
```
Comment (adityav-verma) [inline fabfile.py:8]: Doc string and the function name don't actually match action they perform.

**Review by adityav-verma — CHANGES_REQUESTED** [review]

**Comment by adityav-verma** [comment]
Forked FabPolish and added python3 support and upgraded the packages for coding standard checks.
Travis will require a paid account. Hence, closing this PR.

**Comment by chowmean** [comment]
> Travis will require a paid account. Hence, closing this PR.
Travis is free for open source projects. You can use this https://travis-ci.org/

**Pull request closed without merging**